apps/protocol/lib/generated/lexical/protocol/types/work_done/progress_begin.ex

```diff
@@ -0,0 +1,11 @@
+# This file's contents are auto-generated. Do not edit.
+defmodule Lexical.Protocol.Types.WorkDone.ProgressBegin do
```

Comment:
this mapping, and the end mapping feel a bit wonky. 
Perhaps we change them to `WorkDone.Progress.Begin`,  and`Workdone.Progress.End` ?

It's easy to do, just go edit `type_mappings.json` and re-generate.